templates/telco/osm/onboarding-network-functions.html

```diff
@@ -0,0 +1,191 @@
+    </div>
+		<hr  />
+		<div class="col-12 u-hide--small u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/68971b5f-Onboarding+3.svg",
+        alt="",
+        width="1000",
+        height="500",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+			<h2>VNF onboarding in OSM Hackfests</h2>
+      <p>Canonical regularly participates in the <a href="https://osm.etsi.org/wikipub/index.php/Past_OSM_Hackfests" class="p-link--external">events</a> organized by the OSM community and ETSI’s Centre for Testing and Interoperability. For events like Hackfests and NFV Plugtests Canonical also offers a partner cloud where multiple VNF vendors can deploy and test the interoperability of their Telco network functions. It is also available if VNF vendors want to test their network functions individually before migrating to production environments.</p>
+      <p>52 Participants, More than 100 Network Functions onboarded</p>
+      <p>
+        <a href="https://charmed-osm.com/osm-hackfests" class="p-external--link">
+          Discover our contributions&nbsp;&rsaquo;
+        </a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru-accent is-dark">
+  <div class="row">
+    <div class="col-12">
+      <div class="p-heading-icon--small">
+        <h2 id="get-in-touch">Tell us about your network function</h2>
+        <p>Canonical can help you automate your network function onboarding to Charmed OSM. Please choose the relevant options for your network functions to see how it can work for you.</p>
+        <p><a href="https://charmed-osm.com/#get-in-touch" class="p-button--positive p-external--link">Start now</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-equal-height">
+		<h2>Let us help you in onboarding</h2>
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Charmed OSM deployment support*</h3>
+			<hr />
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Supported with <a href="/ua">Ubuntu Advantage</a></li>
+        <li class="p-list__item is-ticked">Availability of managed and professional services</li>
+        <li class="p-list__item is-ticked">ESM - Security updates for Ubuntu LTS for additional 3-5 years</li>
+      </ul>
+    </div>
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Network function onboarding support </h3>
+			<hr />
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Support for automation of the network function</li>
+        <li class="p-list__item is-ticked">Onboarding it to Charmed OSM</li>
+      </ul>
+    </div>
+    <p>
+      *&nbsp;<a class="p-link--external" href="https://charmed-osm.com/">https://charmed-osm.com/</a>
```

Comment:
Where did this come from? This is not a foot note but a link for the user to learn more.

Reply:
Link in the header, as mentioned by @anasereijo